service/src/main/java/org/exoplatform/extension/exchange/service/util/CalendarConverterUtils.java

```diff
@@ -59,6 +59,7 @@
 
   private static final String             EXCHANGE_EVENT_ID_PREFIX      = "ExcangeEvent";
 
+  private static String                   DEFAULT_TIME_ZONE_EXCHANGE            = "W. Europe";

```

Comment:
You should not use this, because UTC timezone is used on internal Server dates. But for times to store throught EWS, you can use `appointment.setStartTimeZone`, `appointment.getStartTimeZone`, `appointment.getEndTimeZone` and , `appointment.setEndTimeZone`.

Reply:
The purpose of this variable is to save the time zone used to display the event and not the time zone used to save the event in internal exchange server. Let take an example: when demanding to exchange to save an event starting in 00:00 with UTC time zone, the event will be  displayed in Roma (UTC +01) as it starts at 01:00 (although that it was saved in exchange server with UTC time). This is a normal behavior  for simple event but is abnormal for all day events (which should start in 00:00 regardless of the used time zone in exchange calendar).
The idea then is to capture the time zone in which we will display the events in exchange calendar when importing the events from exchange to exo (the first synchronization) and basing to the offset of this time zone , I adjust the start and end times to correctly display the all day events.

Reply:
ok, as discussed, it's for "All day event" which is a particular use case.